src/transform/transform.ts

```diff
@@ -146,12 +154,23 @@ function recursiveAddMarshallingFormat(schema: Schema, marshallingFormat: 'json'
 function processOperationRequests(session: Session<CodeModel>) {
   for (const group of values(session.model.operationGroups)) {
     for (const op of values(group.operations)) {
+      if (op.requests) {
```

Comment:
Nit: Is this check really necessary?

Reply:
Without this check it says that op.requests is possibly undefined

Reply:
I'll add op.requests!.length for the first check and remove the if

Reply:
Huh that's interesting, I wonder when it would ever be undefined (how can you have an operation with no requests....).  If it ends up being problematic can leave it as-is.

Reply:
I removed it for now, I agree that it's unlikely that it would ever return undefined.